client/src/components/FlashCard/CreateFlashcard.js

```diff
@@ -0,0 +1,108 @@
+import React, {useState} from "react"
+import { Button, Stack, TextField } from "@mui/material"
+import axios from 'axios'
+
+const CreateFlashcard = ({ userId, deckId }) => {
+  // how can we use state here to make sure we're validating info
```

Comment:
even if you're inheriting code you shouldn't be committing unnecessary comments and code